src/us/kbase/test/auth2/providers/GoogleIdentityProviderTest.java

```diff
@@ -103,6 +103,7 @@ public void tearDownTest() {
 					new URL("https://glinkredir.com"))
 					.withEnvironment("myenv",
 							new URL("https://mygloginred.com"), new URL("https://myglinkred.com"))
+					.withCustomConfiguration("people-api-host", "https://gpeople.com")
```

Comment:
Is this https://gpeople.com URL supposed to be just be a dummy value? It seems to point to an unregistered domain.

Reply:
Okay, so this is a test, and its just supposed to fail?

Reply:
This static block is just setting up a config for other tests to use. All the urls are dummy values except for the mock server address.